ecl/hqlcpp/hqlhtcpp.ipp

```diff
@@ -58,6 +58,48 @@ private:
     bool grouped;
 };
 
+class GlobalCompareClass
```

Comment:
@shamser I wasn't referring to an existing class - more musing about how I thought it would be structured.

Ideally it would also be combined with the GlobalClassBuilder, but that can wait for another day - these sort classes do not have access to ICodeContext, whereas classes created with the GlobalClassBuilder do.  Using a similar pattern for creating the class would make sense.